src/sst/core/exit.h

```diff
@@ -117,6 +117,8 @@ class Exit : public Action
     bool single_rank;
 };
 
+[[noreturn]] void SST_Exit(int exit_code);
```

Comment:
Not sure if exit.h is the best place for this function to live.  One of the issues is that exit.h is being installed, though it really shouldn't be, and SST_Exit() shouldn't be part of the public API.  The other issue is that even though the names match, the functionality isn't really related to the Exit object.  Of the header files that aren't installed, I think simulation_impl.h is probably the best place for the SST_Exit() function (not as part of the Simulation_impl object, however).  Since The function won't be part of the non-core API, I think it's fine to leave as is for now if you don't want to change it and we can reevaluate later.

Reply:
Okay I can move it anywhere, to prevent it from being publicly exported / documented. The `exit.h` and `exit.cc` seemed logical, even if this function is unrelated to the Exit object.

Reply:
Oh, and since `Exit` is a common name which might conflict with other uses, I used `SST::SST_Exit` instead of `SST::Exit`. If this deviates from normal SST naming conventions, such as case and underscores, it can be renamed. In Rev we use `camelCase` for variables / functions and `CamelCase` for types.